### PR TITLE
NI theme modification

### PR DESCRIPTION
**Pull request by MebinAbraham**
Increased the NI logo dimensions as it was a bit too small.
Also added the ONS logo to the footer.

### How to review?
- Test with `mbs_0204` schema.
- You may need to run `yarn compile`

NI theme looks correct:
1. Header logo should be `280px`
2. ONS logo should be present in the footer.

### CODE REVIEW

**Review by AndrewTorrance — APPROVED** [review]
Difficult to review as there is no definitive acceptance criteria . All I can say is that the images are present , and in the correct position. Footer needs updating later, move images to a cdn comes later

**Comment by armstrongb** [comment]
The alt text on the logo in the header should be amended. It should have the following alt attribute value: **"Department of Finance | An Roinn Airgeadais"**

**Review by armstrongb — CHANGES_REQUESTED** [review]
Header logo alt text amend required.

**Comment by MebinAbraham** [comment]
@armstrongb Sorted alt tag

**Review by armstrongb — APPROVED** [review]

File: app/assets/styles/themes/northernireland/theme.scss [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+
+.logo__img {
+  width: 280px;
+}
+
+.footer_logo {
+  width: 190px;
+  height: auto;
+  margin-left: 1rem;
+}
```
Comment (bennpearson) [inline app/assets/styles/themes/northernireland/theme.scss]: Move this to ` app/assets/styles/themes/northernireland/components/_footer.scss` then imported here with `@import "components/footer"`

File: app/assets/styles/themes/northernireland/theme.scss [hunk 2]
```diff
@@ -1,2 +1,16 @@
 @import "../../themes-common/config";
 @import "../../themes-common/ui";
+
+/**
+ * Theme override
+ */
+
+.logo__img {
+  width: 280px;
+}
```
Comment (bennpearson) [inline app/assets/styles/themes/northernireland/theme.scss]: Move this to ` app/assets/styles/themes/northernireland/components/_logo.scss` then imported here with `@import "components/logo"`

**Review by bennpearson — CHANGES_REQUESTED** [review]

**Review by bennpearson — APPROVED** [review]

**Pull request merged**